diff --git a/fftools-ffi.c b/fftools-ffi.c
new file mode 100644
index 0000000..6cd4258
--- /dev/null
+++ b/fftools-ffi.c
@@ -0,0 +1,6 @@
+#include "fftools-ffi/dart_api.h"
+
+void* a = FFToolsFFIInitialize;
+void* b = FFToolsFFIExecuteFFmpeg;
+void* c = FFToolsFFIExecuteFFprobe;
+void* d = FFToolsCancel;
diff --git a/meson.build b/meson.build
index fb10be5..b85134d 100644
--- a/meson.build
+++ b/meson.build
@@ -26,6 +26,9 @@ libavutil = dependency('libavutil', version: '>= 58.29.100')
 libswresample = dependency('libswresample', version: '>= 4.12.100')
 libswscale = dependency('libswscale', version: '>= 7.5.100')
 
+# fftools-ffi
+libfftools_ffi = dependency('fftools-ffi')
+
 libplacebo = dependency('libplacebo', version: '>=6.338.2',
                 default_options: ['default_library=static', 'demos=false'])
 
@@ -40,7 +43,8 @@ dependencies = [libass,
                 libavutil,
                 libplacebo,
                 libswresample,
-                libswscale]
+                libswscale,
+                libfftools_ffi]
 
 # Keeps track of all enabled/disabled features
 features = {
@@ -265,6 +269,9 @@ sources = files(
     'ta/ta.c',
     'ta/ta_talloc.c',
     'ta/ta_utils.c',
+
+    ## fftools-ffi hack
+    'fftools-ffi.c'
 )
 
 
